docs/source/index.rst

```diff
@@ -44,6 +44,7 @@ DAML SDK documentation
    app-dev/authentication
```

Comment:
You also need to update `docs/configs/pdf/index.rst`.

Reply:
Done in 5ebd3d9 , thanks